c7n/resources/appelb.py

```diff
@@ -760,6 +761,28 @@ def process(self, load_balancers):
                     **args)
 
 
+@AppELB.action_registry.register('modify-security-groups')
+class AppELBModifyVpcSecurityGroups(ModifyVpcSecurityGroupsAction):
+
+    permissions = ("elasticloadbalancing:SetSecurityGroups",)
+
+    def process(self, albs):
+        client = local_session(self.manager.session_factory).client('elbv2')
+        groups = super(AppELBModifyVpcSecurityGroups, self).get_groups(
+            albs)
```

Comment:
small nit, bring this orphan back up to the rest of the function call

Reply:
done